tfjs-layers/src/layers/convolutional.ts

```diff
@@ -706,7 +706,6 @@ serialization.registerClass(Conv3D);
 export class Conv2DTranspose extends Conv2D {
   /** @nocollapse */
   static className = 'Conv2DTranspose';
-  inputSpec: InputSpec[];
```

Comment:
Nice catch. Note to other reviewer: This is declared in the `Layer` interface in `src/engine/topology.ts`.

Reply:
Thank you Matt!